draft-davidson-pp-protocol.md

```diff
@@ -1047,32 +703,34 @@ on the underlying operations in a non-black-box manner. Hence, an
 explicit reduction from the generic VOPRF primitive to the Privacy Pass
 protocol would strengthen these security guarantees.
 
-# Ciphersuites {#pp-ciphersuites}
-
-The Privacy Pass protocol essentially operates as a wrapper around the
-instantiation of the VOPRF that is used in {{voprf-protocol}}. There is
-no extra cryptographic machinery used on top of what is established in
-the VOPRF protocol. Therefore, the ciphersuites that we support are the
-transitively exposed from the underlying VOPRF functionality, we detail
-these below. Each of the ciphersuites is detailed in
-{{I-D.irtf-cfrg-voprf}}.
-
-- VOPRF-P384-HKDF-SHA512-SSWU-RO
-  - maximum security parameter: 192 bits
-- VOPRF-curve448-HKDF-SHA512-ELL2-RO
-  - maximum security parameter: 224 bits
-- VOPRF-P521-HKDF-SHA512-SSWU-RO
-  - maximum security parameter: 256 bits
-
-When referring to the 'maximum security parameter' size above, we are
-referring to the _maximum_ effective key length of the ciphersuite, as
-specified in {{NIST}}. The reason that this is the maximum length is
-because there may be attacks that serve to lower the actual value of the
-security parameter. See {{I-D.irtf-cfrg-voprf}} for more details.
-
-Note than any extension to the Privacy Pass protocol that modifies
-either VOPRF instantiation, or the way that the Privacy Pass API is
-implemented, MUST specify its own ciphersuites.
+# Protocol ciphersuites {#pp-ciphersuites}
+
+The ciphersuites that we describe for the Privacy Pass protocol are
+derived from the core instantiations of the protocol (such as in
+{{voprf-protocol}}).
+
+In each of the ciphersuites below, the maximum security provided
+corresponds to the maximum difficulty of computing a discrete logarithm
+in the group. Note that the actual security level MAY be lower, see the
+security considerations in {{I-D.irtf-cfrg-voprf}} for examples.
+
+## PP(OPRF2)
```

Comment:
Where did "OPRF2" come from?

Reply:
Due to OPRF ciphersuite with ID=0x0002. I could just call it by its full name, but I thought that it introduced naming redundancy.